packages/opentelemetry-context-async-hooks/src/AsyncHooksContextManager.ts

```diff
@@ -252,7 +276,10 @@ export class AsyncHooksContextManager implements ContextManager {
    * @param uid id of the async context
    */
   private _init(uid: number) {
-    this._contexts[uid] = this._contexts[asyncHooks.executionAsyncId()];
+    this._contextRefs.set(
```

Comment:
We could avoid a map entry for `this._contextRefs.get(asyncHooks.executionAsyncId() == undefined`

Reply:
Added a if for that